docker-compose.yml

```diff
@@ -80,7 +80,20 @@ services:
       - "9411:9411"
     networks:
       - otlp-network
-  
+
+  prometheus:
+    image: prom/prometheus
```

Comment:
can you pin the image to some current version, so it will not break at some point in time due to some compatibility changes?

Reply:
@zimny changes pushed